PyTables/PyTables
# Release 3.2.3

This an attempt to prepare release 3.2.3

I cherry-picked all bugfixes from `develop` and anything that seemed minor enough not to break things. I have pasted the release notes below. The current PR passes the `--heavy` tests.

I skipped large PRs such as #493, #519, #541 and #543. 

ToDo:
 - [x] ~~Make a PR of #553 to `master` and~~ make sure it gets in this release.
 - [x] cherry-pick #559 c-blosc 1.9.2
 - [x] try to get #561 in.
 - [x] handle #539  (`test_indexvalues.py` fails on python 3.5)
 - [x] ~~handle #545 (HDF5 1.10 break pytables)~~ up to package maintainers.
 - [x] Double check the release notes
 - [x] Run `--heavy` tests (after #553 is merged)
 - [x] set `VERSION` to 3.2.3 

```
Improvements
------------

- It is now possible to use HDF5 with the new shared library naming scheme
  (>= 1.8.10, hdf5.dll instead of hdf5dll.dll) on Windows (:issue:`540`).
  Thanks to Tadeu Manoel.
- Now :program: `ptdump` sorts output by node name and does not print a
  backtrace if file cannot be opened.
  Thanks to Zbigniew Jędrzejewski-Szmek.


Bugs fixed
----------

- Only run `tables.tests.test_basics.UnicodeFilename` if the filesystem
  encoding is utf-8. Closes :issue:`485`.
- Add lib64 to posix search path. (closes :issue:`507`)
  Thanks to Mehdi Sadeghi.
- Ensure cache entries are removed if fewer than 10 (closes :issue:`529`).
  Thanks to Graham Jones.
- Fix segmentation fault in a number of test cases that use
  :class:`index.Index` (closes :issue:`532` and :issue:`533`).
  Thanks to Diane Trout.
- Fixed the evaluation of transcendental functions when numexpr is
  compiled with VML support (closes :issue:`534`, PR #536).
  Thanks to Tom Kooij.
- Make sure that index classes use buffersizes that are a multiple
  of chunkshape[0] (closes :issue:`538`, PR #538).
  Thanks to Tom Kooij.
- Ensure benchmark paths exist before benchmarks are executed (PR #544).
  Thanks to rohitjamuar.
```
